### ISSUE TITLE
How can one deploy using GitHub action

### ISSUE THREAD

**Issue by sid-r-singh** [poanchen/poanchen.github.io#28]
Hey your blog is just amazing. But as you know, now with the help of GitHub action there is no need of any other 3rd party CI/CD like Azure functions. Can you share the relevant workflow as to how to deploy using GitHub action.

**Comment by poanchen** [comment]
I am glad that you like it. I do use Azure DevOps as CI/CD to handle build/deployment. Are you saying you want the GitHub Action to handle it instead?

**Comment by sid-r-singh** [comment]
Yes, I wish to use github action instead of Azure since it come free with GitHub.

**Comment by poanchen** [comment]
@sid-r-singh did you end up using github action to deploy website now?
